core/scss/components/button-group/module.scss

```diff
@@ -1,15 +1,38 @@
-@import '../../tools/functions/core';
+@import '../../tools/mixins/core';
 
-@import '../../colors/settings';
-@import '../../settings/core';
-@import '../../settings/media';
-@import '../../typography/settings';
-@import '../button/settings';
-@import '../../tools/functions/core';
+// Settings
+@import '../../settings/settings';
 
+// Tools
 @import '../../tools/mixins/border-radius';
+@import '../../tools/mixins/flex';
+@import '../../tools/mixins/focus.scss';
+
+// Generic
+@import '../../generic/global';
+@import '../../generic/normalize.scss';
+
+
+// Tools
+@mixin brand-font-face-regular {
```

Comment:
This mixin already exists in the typography/mixins.scss file. No need to recreate it.

Reply:
got it